File: server/ruby/app/hosted_tokenization/hosted_tokenization_service.rb
class HostedTokenizationService
  def self.init_hosted_tokenization()
    merchant_client = MerchantClientConfig.merchantClient()

    hosted_tokenization_client = merchant_client.hosted_tokenization()

    return hosted_tokenization_client.create_hosted_tokenization(CreateHostedTokenizationRequest.new)
  end
end

File: server/python/hosted_tokenization/hosted_tokenization_service.py
from configuration.merchant_client_config import merchant_client
from onlinepayments.sdk.domain.create_hosted_tokenization_request import CreateHostedTokenizationRequest # type: ignore

def init_hosted_tokenization():
    hosted_tokenization_request = CreateHostedTokenizationRequest()
    return merchant_client().hosted_tokenization().create_hosted_tokenization(hosted_tokenization_request)